test/acceptance/tests/example/example_test.go

```diff
@@ -1,3 +1,4 @@
+// todo: rename package
```

Comment:
Added these for the templates

Reply:
Curious why are you thinking to rename it?

Reply:
> Haha no this is what the person copying it needs to do. Maybe I could make
that clearer?
Oh, I see 😄  Hmmm, maybe this could be a full sentence saying something like "Rename package to your test package."